build.sbt

```diff
@@ -139,7 +139,7 @@ lazy val hashing = module(project, "hashing")
 lazy val io = module(project, "io")
   .dependsOn(aws, testkit % Test)
   .settings(
-    crossScalaVersions := List(Versions.Scala212, Versions.Scala213),
+    crossScalaVersions := List(Versions.Scala212, Versions.Scala213, Versions.Scala3),
```

Comment:
As discussed offline, we can probably [ditch 2.12](https://github.com/adzerk/apso/issues/812), if you'd like to opportunistically handle that in another PR.